net-dns/maradns/maradns-3.5.0005.ebuild

```diff
@@ -17,12 +16,17 @@ SLOT="0"
 KEYWORDS="amd64 ~mips ~ppc x86"
 IUSE="examples ipv6"
 
-DEPEND="${PYTHON_DEPS}"
-RDEPEND="${DEPEND}
-	acct-user/maradns
+BDEPEND="${PYTHON_DEPS}"
+DEPEND="
 	acct-group/maradns
 	acct-user/duende
+	acct-user/maradns
 	dev-lang/perl"
```

Comment:
maybe double check, I'm willing to bet perl should actually go into `BDEPEND`

Reply:
Yeah, that was my first thought as well, looks like it's BDEPEND